### ISSUE TITLE
X::TypeCheck::Argument could use a minor wording tweak

### ISSUE THREAD

**Issue by zoffixznet** [rakudo/rakudo#1747]
```
19:38 	Zoffix 	m: proto z($, $?) {*}; z 1, 2, 3
19:38 	camelia 	rakudo-moar 9e6ae276a: OUTPUT: «5===SORRY!5=== Error while compiling <tmp>␤Calling z(Int, Int, Int) will never work with proto signature ($, $?)␤at <tmp>:1␤------> 3proto z($, $?) {*}; 7⏏5z 1, 2, 3␤»
```

The above message reads "proto signature" [which suggests](https://irclog.perlgeek.de/perl6/2018-04-18#i_16062597) some special signature.

It might be clearer to reword that to "signature of the proto"

**Comment by lizmat** [comment]
Fixed with 0de80523c56cf7d89dba79a4 .  Not sure tests for this would be needed.

**Comment by zoffixznet** [comment]
> Not sure tests for this would be needed.

Thanks. I think it's fine to close without any.
